Dockerfile.hub

```diff
@@ -21,11 +21,12 @@ RUN wget -q https://repo.continuum.io/miniconda/Miniconda3-4.2.12-Linux-x86_64.s
 
 RUN /opt/conda/bin/conda install --yes -c conda-forge \
     python=3.5 \
-    configurable-http-proxy \
-    notebook \
-    jupyterhub \
+    configurable-http-proxy=3.0.0 \
```

Comment:
The conda recipe for jupyterhub includes configurable-http-proxy, so we should probably shouldn't list it here explicitly and let the jupyterhub package decide what version it needs.